tests/_client.lua

```diff
@@ -32,8 +32,19 @@ local function open_window(class, title, options)
         }
         window:set_geometry_hints(nil, geom, Gdk.WindowHints.RESIZE_INC)
     end
+    if options.maximize_before then
+        window:maximize()
+    end
```

Comment:
No, neither GTK nor Qt support it in their `GtkWindow`/`QWindow` classes. It would require a custom C client to trigger such code path, but it would be a limited (read: none) value as no client I know of ever trigger partial maximization by themselves.